sources/Renderer/OpenGL/Platform/MacOS/MacOSGLSwapChainContext.mm

```diff
@@ -30,7 +30,6 @@
     return MacOSGLSwapChainContext::MakeCurrentNSGLContext(static_cast<MacOSGLSwapChainContext*>(context));
 }
 
-
```

Comment:
There are some probably unintentional changes remaining in these files. You can simply revert them by checking out the source from the upstream repo.